settings/browsersettings.cpp

```diff
@@ -13,7 +13,9 @@
 #include <QXmlStreamReader>
 #include "browsersettings.h"
 
-static const QString openSearchPath("/usr/lib/mozembedlite/chrome/embedlite/content/");
+static const QStringList gOpenSearchPathList = QStringList() \
+                                            << QString("/usr/lib/mozembedlite/chrome/embedlite/content/") \
+                                            << QDir::homePath() + "/.local/share/org.sailfishos/sailfish-browser/opensearch/";
```

Comment:
Yeap, for sure. Would be be having a define in common.pri (one place) and using that define.